### PR TITLE
1.2.1 release of Deploy

### PR DESCRIPTION
**Pull request by ukclivecox**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -76,14 +77,14 @@ docker-push:
 	docker push ${IMG}
 
 #password can be found at https://connect.redhat.com/project/4805411/view
-redhat-image-scan: docker-build docker-push
+redhat-image-scan: #docker-build docker-push
```
Comment (ryandawsonuk) [inline Makefile]: @cliveseldon is the comment here deliberate? Fine if you want to do it that way, just not sure if it's deliberate

File: README.md [hunk 2]
```diff
@@ -10,6 +10,12 @@ There are dependencies needed for running Deploy and marketplace imposes restric
 
 ## Building
 
+Dependencies
+
+ * [Operator SDK](https://sdk.operatorframework.io/)
+   * Tested with 1.8.0
```
Comment (ryandawsonuk) [inline README.md:16]: Worth noting there is also a line at the very beginning that says `Built with operator-sdk (v1.7.2)`

File: bundle.Dockerfile [hunk 3]
```diff
@@ -15,11 +15,6 @@ LABEL operators.operatorframework.io.metrics.project_layout=helm.sdk.operatorfra
 LABEL operators.operatorframework.io.test.mediatype.v1=scorecard+v1
 LABEL operators.operatorframework.io.test.config.v1=tests/scorecard/
 
-#seems these have to be put in manually but bundle scan fails without them
-LABEL com.redhat.openshift.versions="v4.7"
-LABEL com.redhat.delivery.operator.bundle=true
-LABEL com.redhat.delivery.backport=false
```
Comment (ryandawsonuk) [inline bundle.Dockerfile:21]: I guess we'll find out very soon whether this is a problem or not

**Review by ryandawsonuk — APPROVED** [review]
Have made some comments but /lgtm

File: Makefile (hunk 1 above)
Comment (ukclivecox) [reply Makefile]: no its a typo from testing. will fix.

File: README.md (hunk 2 above)
Comment (ukclivecox) [reply README.md:16]: will fix

**Pull request merged**